Add script tags to .js file

BoatrightTBC opened the pull request:
Custom Content Creator injects the BARE CONTENTS of the .js file, not a link to it. As a result, the javascript is not wrapped in script tags and does not execute. So I put script tags in the .js file.

dedevillela commented:
Hi BoatrightTBC, and thanks for pointing that out.

For your knowledge, the code does not contain the <script> tags for CI validation purposes.

By the way, I have added instructions to the Readme file so that the plugin works properly.

Best regards,

Andre Villela

BoatrightTBC commented:
OK, for my sake (and you can close this if you like and ignore it) I created source and release folders with two versions of the JS. 

I also updated the readme to reflect that rather than trusting people to edit source code, and fixed a few grammer and formatting issues in the readme. 

I'll persist that branch of my repo so I can link to it whether you merge or not. 

Thanks for the quick reply.

dedevillela commented:
Hi BoatrightTBC, and sorry for the delay! I'd love to merge you pull request, but the Codacy checks revealled some issues with code standards, mostly can be resolved with comment placement for variable declarations. There's also conflicts with the README and the script file; once its all good I'll proceed with the PR. Thanks again for your input, time and effort while contributing to this project!

Kind regards,
